.github/workflows/unit-testing-and-coverage.yml

```diff
@@ -53,7 +51,9 @@ jobs:
             "print(config['flake8_plugins']['deps']);"
           )
           pip install "flake8>=5.0.4" $(python -c "${grab_f8_plugins[*]}")
-          python -m flake8 aiosmtpd setup.py housekeep.py release.py
+          echo "Running flake8 ... there will be no output if no errors:"
+          python -m flake8 aiosmtpd housekeep.py release.py
+          echo "flake8 done."
```

Comment:
Added just to ensure that flake8 actually runs. flake8's silence on no error sometimes make me wonder if it ran or not 😅

Reply:
FWIW it's quite common to wrap linters with the pre-commit.com too.